cluster/swarm/cluster.go

```diff
@@ -220,8 +220,12 @@ func (c *Cluster) createContainer(config *cluster.ContainerConfig, name string,
 
 	c.scheduler.Unlock()
 
-	log.WithFields(log.Fields{"NodeName": n.Name, "NodeID": n.ID}).Debugf("Scheduling container %s to ", name)
 	container, err := engine.CreateContainer(config, name, true, authConfig)
+	containerFlag := name
+	if containerFlag == "" && err == nil {
+		containerFlag = stringid.TruncateID(container.ID)
+	}
+	log.WithFields(log.Fields{"NodeName": n.Name, "NodeID": n.ID}).Debugf("Scheduling container %s to ", containerFlag)
```

Comment:
I am afraid that success or failure should be divided into two different kinds. Error, failed to .......; Seccess, scheduled to node.....(while not scheduling)
My own opinion.
@xianlubird